### ISSUE TITLE
Consider update on your install.sh script

### ISSUE THREAD

**Issue by dkalog** [intrig-unicamp/mininet-wifi#157]
Hi *,
Please consider in mininet-wifi deps checking the mac80211 kernel module  otherwise installing
linux-image-extra-$(uname -r) and a reboot.  You may update the instructions in the Readme.md

On stock ubuntu xenial64  it worked.

**Comment by ramonfontes** [comment]
Isn't mac80211_hwsim compiled by default?

**Comment by dkalog** [comment]
I am use the vagrant image "ubuntu/xenial64" which is the stock image from canonical to produce a recipe. So unless there is a mishap (I wouldn't be surprised ... ) it is not builtin.  As a matter of fact this the case in the  GCP also...  :-/..

**Comment by ramonfontes** [comment]
Can `sudo apt-get install linux-image-extra-'uname -r'` solve the issue on both cases?

**Comment by dkalog** [comment]
Yes it can solve it..but you need a reboot also ... I can provide a vagrant image as a base image if it helps..

**Comment by ramonfontes** [comment]
I will add a pointer in the user manual.
